nrf_security/doc/backend_config.rst

```diff
@@ -336,8 +518,63 @@ ECC core support can be enabled by setting the :option:`CONFIG_MBEDTLS_ECP_C` Kc
 
 .. note::
    * For features provided with :ref:`Choice<nrf_security_backend_config_multiple>` support, the enabled backend that is first in order is selected by default.
-   * The :ref:`nrf_oberon_readme` only supports ECC curve secp256r1.
 
+Feature support
+===============
+
++-----------+-------------------+-------------+------------+
+| Algorithm | Backend           | Curve group | Curve type |
++===========+===================+=============+============+
+| ECP       | cc3xx             | NIST        | secp192r1  |
+|           |                   |             +------------+
+|           |                   |             | secp224r1  |
+|           |                   |             +------------+
+|           |                   |             | secp256r1  |
+|           |                   |             +------------+
+|           |                   |             | secp384r1  |
+|           |                   |             +------------+
+|           |                   |             | secp521r1  |
+|           |                   +-------------+------------+
+|           |                   | Koblitz     | secp192k1  |
+|           |                   |             +------------+
+|           |                   |             | secp224k1  |
+|           |                   |             +------------+
+|           |                   |             | secp256k1  |
+|           |                   +-------------+------------+
+|           |                   | Curve25519  | Curve25519 |
+|           +-------------------+-------------+------------+
+|           | nrf_oberon        | NIST        | secp256r1  |
+|           |                   |             +------------+
+|           |                   |             | secp224r1  |
+|           |                   +-------------+------------+
+|           |                   | Curve25519  | Curve25519 |
+|           |                   |             +------------+
+|           |                   |             | Ed25519    |
```

Comment:
@frkv

Reply:
Ah yes.  Needs to be corrected. Is only available in PRs in mbed TLS at this moment